backend/test/activitypub/follow.spec.ts

```diff
@@ -24,7 +24,7 @@ describe('ActivityPub', () => {
 			globalThis.fetch = async (input: any) => {
 				if (input.url === `https://${domain}/ap/users/sven2/inbox`) {
 					assert.equal(input.method, 'POST')
-					const data = await input.json()
+					const data = await (input as Response).json()
```

Comment:
`input` is actually a `Request` not `Response` 😱 

But I think the correct fix here is to give the function being defined a proper type:

```
			globalThis.fetch = async (input: RequestInfo) => {
				const request = new Request(input)
				if (request.url === `https://${domain}/ap/users/sven2/inbox`) {
					assert.equal(request.method, 'POST')
					const data = await request.json()
					receivedActivity = data
					console.log({ receivedActivity })
					return new Response('')
				}

				throw new Error('unexpected request to ' + request.url)
			}
```

Reply:
sorry! 😱 😭 

as I scanned thought the code and saw the `.json` I instinctively thought it was a response 😓
(I've checked and I've done that in a couple of places I've fixed all of them 😓)

By the way, awesome solution, I tend to forget that you can pass the request request infos in that way 😃 🤩